lib/logidze/ignore_log_data.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Logidze
+  # Add `has_logidze` method to AR::Base
+  module IgnoreLogData
+    extend ActiveSupport::Concern
+
+    module BackportIgnoredColumns # :nodoc:
```

Comment:
Let’s move this patch into a separate file and add `columns` information manipulation (like in the original patch for Rails 5 or in ignorable).